src/test/resources/junit-platform.properties

```diff
@@ -0,0 +1,2 @@
+junit.jupiter.execution.parallel.enabled=TRUE
```

Comment:
This makes it to run tests in parallel on N threads
https://junit.org/junit5/docs/current/user-guide/#writing-tests-parallel-execution

Reply:
Nice. This way we have to explicitly flag our tests that require SAME_THREAD...which probably marks them as good refactoring opportunities.